### PR TITLE
Add SageMaker Jumpstart notebook for multi-category SEC filing modeling

### PR DESCRIPTION
**Pull request by sophiayue1116**
*Description of changes:*
Add SageMaker Jumpstart notebook for multi-category SEC filing modeling using:
1. [smjsindustry](https://pypi.org/project/smjsindustry/) package for nlp scoring calculation
1. [AutoGluon image](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#autogluon-training-containers) for training and inference

*Testing done:*
local black-nb test

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sagemaker-jumpstart/multicategory_sec/code/config.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# AutoGluon Predictor constructor arguments
+# - see https://github.com/awslabs/autogluon/blob/master/tabular/src/autogluon/tabular/predictor/predictor.py#L51-L159
+ag_predictor_args:
+  label: industry_code
+
+# AutoGluon Predictor.fit arguments
+# - see https://github.com/awslabs/autogluon/blob/master/tabular/src/autogluon/tabular/predictor/predictor.py#L250-L590
```
Comment (jkroll-aws) [inline sagemaker-jumpstart/multicategory_sec/code/config.yaml:7]: Take another look at the line number links, they look a little off. The problem with linking to the latest version is that line numbers will shift over time. See what you think - it's okay to leave as-is.

**Comment by sophiayue1116** [comment]
<p>Sure, I will change to MNIST.</p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sophiayue1116** [comment]
<p>Good catch, thanks</p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sophiayue1116** [comment]
<p>This is the internal name of the product and used by PM for tracking purpose. </p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sophiayue1116** [comment]
<p>sure!</p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: sagemaker-jumpstart/multicategory_sec/code/config.yaml (hunk 1 above)
Comment (sophiayue1116) [reply sagemaker-jumpstart/multicategory_sec/code/config.yaml:7]: Got it, I will remove the lines in the link.

**Comment by sophiayue1116** [comment]
<p>In the jumpstart studio, customer does not need to select the kernel type and the studio notebook picks up a predefined kernel type. This notebook is customized for classic notebook users and we have the embedded notebook in the jumpstart studio. </p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sophiayue1116** [comment]
<p>I checked the total volume used is 2.1G, the default size should be enough. I can remove this line if it causes confusion. </p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by sophiayue1116** [comment]
<p>I am not familiar with the use case. The NLPScorer API is built on top of sagemaker processing job. Those args are from Processor. Should I just remove these 2 lines?</p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by jkroll-aws** [comment]
<p>Fine to leave as-is. I didn't read carefully and thought there were multiple files of 1 GB. What you wrote is clear, and agreed that the default volume size is sufficient then.</p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by jkroll-aws** [comment]
<p>I think it's helpful to show that a KMS key can be passed. How about just revise the comments slightly so it's a bit clearer that the customers can encrypt their data if needed - </p><p> </p><p><code>KMS key ID to encrypt the processing volume</code></p><p> </p><p><code>KMS key ID to encrypt processing job outputs</code>  </p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by jkroll-aws** [comment]
<p>Okay, just be aware that this notebook can still be opened in Studio as well.</p>

---

 <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3006/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by jkroll-aws — APPROVED** [review]

**Pull request merged**